acceptance/suites/pre_suite/foss/70_install_puppet.rb

```diff
@@ -63,7 +63,7 @@ def install_puppet_from_msi( host, opts )
   end
 
 step "Upgrade nss to version that is hopefully compatible with jdk version puppetserver will use." do
-  nss_package=nil
+  nss_package_name=nil
```

Comment:
In case anyone cares, `case` in ruby does not create a new scope. Additionally, as ruby sees/parses variables it will initialize them, so this line isn't needed at all.

Reply:
This line isn't needed but I (at least) find Ruby's behavior surprising and think this change is a good one.

Reply:
I'd be fine without the line but if y'all want it I'm cool with that too